docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/01 - Identifiers.md

```diff
@@ -0,0 +1,87 @@
+# Identifiers
+
+## Overview
+By default LORIS provides 2 different identifiers for each candidate: 
+
+- **CandID**, also known as the **DCCID**, is a unique randomized 6-digit numeric ID (e.g. '436792') assigned automatically by LORIS upon the candidate's registration. The CandID is not configurable.
+- **PSCID** (Project Study Center ID) is a unique configurable ID. It can be set up to be either manually entered when registering a candidate or automatically generated and usually contains the site or project abbreviation followed by sequential or randomized characters (e.g. 'MTL0006'), but its exact format is customizable.
+
+The format and the generation of the **PSCID** must be configured by an admin at the setup stage.
+
+## Configuration
+
+### Configuration from the front end
+_not yet available_
+
+### Configuration from SQL
+_not yet available_
+
+### Configuration from the config.xml file
+
+Both the format and the generation of PSCIDs can be configured by an administrator in the `config.xml` file of the `%LORIS_ROOT%/project/` directory. these settings are applied to any and all new candidates.
+
+PSCIDs can be created for new subjects in one of 3 ways: *sequentially generated*, *manually entered*, or *randomly generated*.
+
+1. ***sequential*** generates PSCIDs sequentially for each new candiddate registered. **(default)**
+
+ ```xml
+ <PSCID>
+     <generation>sequential</generation> 
+     <structure>
+         <seq type="siteAbbrev"/>
+         <seq type="numeric" length="4" min="10" max="9999"/>
+     </structure>
+ </PSCID>
+ ```
+ > Example PSCID generated: MTL1234
+ > Where the site's alias is MTL
+
+2. ***manual*** asks the user to enter the PSCID when registering a new candidate.
+
+ ```xml
+ <PSCID> 
+     <generation>user</generation> 
+     <structure>
+         <seq type="alphanumeric" length="2"/>
+     </structure>
+ </PSCID>
+ ```
+  > Example PSCID generated: A1
+
+3. ***random*** generates PSCIDs with a random numerical value for each new participant registered.
+
+ ```xml
+ <PSCID>
+     <generation>random</generation> 
+     <structure>
+         <seq type="static">PROJ</seq>
```

Comment:
I feel like this example is a little ambiguous given that there's now also `projectAbbrev` option.
```suggestion
         <seq type="static">PREFIX</seq>
```

or maybe `STUDY`? Something that makes it more different from the Project Alias option.

The example comment on 62 would need to be updated too.

Reply:
done